handlers/src/main/java/com/nextdoor/bender/handler/dynamodb/DynamodbInternalEvent.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * Copyright 2017 Nextdoor.com, Inc
+ *
+ */
+
+package com.nextdoor.bender.handler.dynamodb;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.LinkedHashMap;
+
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent.DynamodbStreamRecord;
+import com.google.gson.stream.JsonWriter;
+import com.nextdoor.bender.InternalEvent;
+import com.nextdoor.bender.LambdaContext;
+
+public class DynamodbInternalEvent extends InternalEvent {
+    public static final String DYNAMODB_KEYS = "__keys__";
+    private DynamodbStreamRecord record;
+
+    public DynamodbInternalEvent(DynamodbStreamRecord record, LambdaContext context)
```

Comment:
Modify this to be `public DynamodbInternalEvent(DynamodbStreamRecord record, String stringRecord, LambdaContext context)` and just do `super(stringRecord, context, ...)`. Note, don't do serialization in here because how the JDK handles exceptions in constructors.